packages/dendron-next-server/styles/scss/support/_variables.scss

```diff
@@ -0,0 +1,135 @@
+//
```

Comment:
I am confused about this second `variables.scss` file. What does the `support` folder imply?